pkg/controller/hyperconverged/hyperconverged_controller_test.go

```diff
@@ -1742,6 +1742,116 @@ var _ = Describe("HyperconvergedController", func() {
 				})
 
 			})
+
+			Context("Remove old metrics services and endpoints", func() {
+				services := []string{
+					"hyperconverged-cluster-operator-metrics",
+					"hyperconverged-cluster-webhook-metrics",
+				}
+
+				serviceEvents := make([]commonTestUtils.MockEvent, 0)
+				for _, service := range services {
+					serviceEvents = append(serviceEvents, commonTestUtils.MockEvent{
+						EventType: corev1.EventTypeNormal,
+						Reason:    "Killing",
+						Msg:       fmt.Sprintf("Removed %s Service", service),
+					})
+
+				}
```

Comment:
It will work. I wonder if it won't be simpler just to add these two services as a golang structs.